VikingQuiz.UI/src/Components/FormComponent/FormInput/FormInput.tsx

```diff
@@ -3,10 +3,10 @@ import './FormInput.css';
 
 const formInput: any = (props: any) => {
     return (
-       <div className="formgroup col-xs-12">
-        <label className="formlabel" htmlFor={props.InputId}>{props.InputLabel}</label>
-        <input id={props.InputId} name={props.name} className="myinput form-control" type={props.InputType} onChange={props.changed}/>
-        <div className="formerror">{props.ErrorMessage}</div>
+       <div className="form-group col-xs-12">
+        <label className="form-label" htmlFor={props.InputId}>{props.InputLabel}</label>
+        <input id={props.InputId} name={props.name} className="my-input form-control" type={props.InputType} onChange={props.changed}/>
```

Comment:
The class my-input doesn`t look good. Please rename it to something more meaningful